generators/app/templates/src/app/components/SearchBar/index.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import SearchBar from './index';
+
+describe('#SearchBar', () => {
+  let component = null;
+  const handleSubmit = jest.fn();
+
+  beforeAll(() => {
+    component = mount(
+      <SearchBar submitButtonText="Search" onSubmit={handleSubmit}>
+        <input />
+      </SearchBar>
+    );
+  });
+
+  afterAll(() => {
+    component.unmount();
+  });
```

Comment:
I think we should change this to a `beforeEach` and `afterEach` methods. Mounting the component once and not unmounting it after each test makes each test depend on the previous one execution.